hat/constants.py

```diff
@@ -1 +1,7 @@
 DATE_FORMAT = "%Y-%m-%d"
+
+ACCOUNT_FEATURE_FLAGS = [
+    {"code": "SHOW_HOME_ONLINE", "name": "Display plugin online home page"},
+    {"code": "SHOW_BENEFICIARY_TYPES_IN_LIST_MENU", "name": "Display beneficiaries by types in side menu"},
+    {"code": "SHOW_PAGES", "name": "Show page menu"},
+]
```

Comment:
```suggestion
```

I wouldn't write these values as constants because they are already stored in DB.